Lib/tkinter/test/test_tkinter/test_widgets.py

```diff
@@ -936,7 +936,7 @@ def test_digits(self):
 
     def test_from(self):
         widget = self.create()
-        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=float_round)
+        self.checkFloatParam(widget, 'from', 100, 14.9, 15.1, conv=float_round, allow_not_rounded=True)
```

Comment:
Does not help to specify other value for the `conv` argument: `noconv` or `float`? It can be version-dependent, e.g. `conv=float_round if get_tk_patchlevel() < (8, 6, 10) else noconv`.

Reply:
I am not sure if this is going to be corrected, though (I am thinking of reporting it to the Tcl team so they can decide whether it is intentional or a bug). If it is going to be corrected by the team, we will need another PR to update the patchlevel range for which `noconv` is used, but I assume this is fine with you (thank you: I will push this recommendation).